src/__forks__/createRequestError.js

```diff
@@ -0,0 +1,10 @@
+import formatRequestErrors from './formatRequestErrors';
+
+export default function createRequestError(request, requestType, payload) {
```

Comment:
Yes, it is based on createRequestError from https://github.com/facebook/relay/blob/master/src/network-layer/default/RelayDefaultNetworkLayer.js#L199